recipes-domu/domu-image-weston/domu-image-weston.bbappend

```diff
@@ -0,0 +1,70 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+FILESEXTRAPATHS_prepend := "${THISDIR}/../../inc:"
+FILESEXTRAPATHS_prepend := "${THISDIR}/../../recipes-domx:"
+
+do_configure[depends] += "domd-image-weston:do_domd_install_machine_overrides"
+
+SRC_URI = " \
+    repo://github.com/iartemenko/manifests;protocol=https;branch=pr_testv3;manifest=prod_gen3_test/domu.xml;scmdata=keep \
```

Comment:
I thought, that should be located in xen-troops project...

Reply:
good catch, will update